src/components/Book/Book.tsx

```diff
@@ -6,13 +6,16 @@ import { useLocation, useNavigate } from 'react-router-dom';
 import BookModel from 'models/BookModel';
 import axios from 'axios';
 import BookDialog from 'components/BookDialog/BookDialog';
+import tokenService from 'services/tokenService';
 
 const Book = () => {
   const [book, setBook] = useState<BookModel>({} as BookModel);
   const [modify, setModify] = useState<Boolean>(false);
   const location = useLocation();
   const id = parseInt(location.pathname.split('/')[2]);
   const navigate = useNavigate();
+  const { isRoleAdmin } = tokenService;
+  const isAdmin = isRoleAdmin();
```

Comment:
Is there a way to write this in one line, e.g. `const isAdmin = tokenService.isRoleAdmin();`